core/Piranha.Manager/Areas/Manager/Views/Shared/EditorTemplates/ImageField.cshtml

```diff
@@ -8,7 +8,7 @@
     <div class="col-xs-9">
         @Html.HiddenFor(m => m.Id)
         <div class="well well-sm">
-            <a id="@(prefix)_Filename" href="#" data-toggle="modal" data-target="#modalImgPreview" data-filename="@(Model.Media != null ? Model.Media.Filename : "")" data-url="@Url.Content(Model.Media != null ? Model.Media.PublicUrl : "")" data-contenttype="@Model.Media?.ContentType" data-filesize="@(Math.Round(Model.Media != null ? Model.Media.Size : 0 / (double)1024, 1))kB" data-modified="@Model.Media?.LastModified.ToString("yyyy-MM-dd")">@(Model.Media != null ? Model.Media.Filename : "")</a>
+            <a id="@(prefix)_Filename"  href="#" data-toggle="modal" data-target="#modalImgPreview" data-filename="@(Model.Media != null ? Model.Media.Filename : "")" data-url="@Url.Content(Model.Media != null ? Model.Media.PublicUrl : "")" data-contenttype="@Model.Media?.ContentType" data-filesize="@(Math.Round(Model.Media != null ? Model.Media.Size : 0 / (double)1024, 1))kB" data-modified="@Model.Media?.LastModified.ToString("yyyy-MM-dd")">@(Model.Media != null ? Model.Media.Filename : "")</a>

```

Comment:
Am I missing something? It looks like you've only added an extra whitespace here!

Best regards

Håkan